### DISCUSSION TITLE
Collapsible list item in GitHub-flavored Markdown

### DISCUSSION THREAD

**Discussion by ero-qt** [community/community#52738] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is the titular feature achievable within GitHub's Markdown pages?

---
An HTML-only example of what I'm talking about:

<details open><summary>Parent1</summary>

<ul>
  <li>Child1.1</li>
  <li>
    <details><summary>Child1.2</summary>
      <ul>
        <li>Child1.2.1</li>
        <li>Child1.2.2</li>
      </ul>
    </details>
  </li>
  <li>Child1.3</li>
</ul>

</details>

> <details><summary>(Code)</summary>
> 
> ```html
> <details open><summary>Parent1</summary>
> 
> <ul>
>   <li>Child1.1</li>
>   <li>
>     <details><summary>Child1.2</summary>
>       <ul>
>         <li>Child1.2.1</li>
>         <li>Child1.2.2</li>
>       </ul>
>     </details>
>   </li>
>   <li>Child1.3</li>
> </ul>
> 
> </details>
> ```
> </details>

---
My best attempt using Markdown bullet points:

<details open><summary>Parent1</summary>

* Child1.1 <details><summary>Child1.2</summary>
    * Child1.2.1
    * Child1.2.2
  </details>
* Child1.3

</details>

> <details><summary>(Code)</summary>
> 
> ```html
> <details open><summary>Parent1</summary>
> 
> * Child1.1 <details><summary>Child1.2</summary>
>     * Child1.2.1
>     * Child1.2.2
>   </details>
> * Child1.3
> 
> </details>
> ```
> </details>

---

You can see that the Markdown is super sketchy and hacky and is missing a bullet point on `Child1.2`. You can also see that there is some extra spacing between `Child1.2` and the `Child1.3` list item in both examples.

Is what I want achievable? This is for a custom wiki sidebar.

**Comment by BingeCode** [comment]
I tried to achieve the same with some html/css hacking, but your approach seems cleaner. 
I basically styled the details element as a regular li element without the bullet point and positioned it absolutely in relation to the list.
Check out [my code pen](https://codepen.io/BingeCoding/pen/rNrdbaE)

I will try your approach to make a mixed unordered / collapsible list. tbc
